docker-compose.yaml

```diff
@@ -1,6 +1,7 @@
 version: "3.9"
 services:
   da:
+    container_name: da-celestia
```

Comment:
what's the purpose of adding the container name (other than for labeling reasons)?

Reply:
ah nvm, I see why

Reply:
🫡